### PR TITLE
improve the codeceptjs info

### PR DESCRIPTION
**Pull request by PeterNgTr**
## Motivation/Description of the PR

Applicable helpers:

- [ ] Webdriver
- [ ] Puppeteer
- [ ] Nightmare
- [ ] REST
- [ ] Appium
- [ ] Protractor
- [ ] TestCafe

This gives more info when using `npx codeceptjs info`

```sh
$npx codeceptjs info

 Environment information:-

codeceptVersion:  "2.3.0"
nodeInfo:  10.15.0
osInfo:  macOS Mojave 10.14.6
cpuInfo:  (8) x64 Intel(R) Core(TM) i7-6820HQ CPU @ 2.70GHz
chromeInfo:  78.0.3904.108
edgeInfo:  N/A
firefoxInfo:  65.0.1
safariInfo:  13.0.4
helpers:  {
 "WebDriver": {
  "url": "https://www.amazon.de",
  "browser": "chrome",
  "desiredCapabilities": {
   "chromeOptions": {
    "args": [
     "--headless",
     "--disable-gpu",
     "--window-size=1440,700"
    ]
   }
  },
  "windowSize": "1440x700"
 }
}
plugins:  "You don't have any enabled plugins"
***************************************

If you have questions ask them in our Slack: shorturl.at/cuKU8
    
Or ask them on our discussion board: https://codecept.discourse.group/
    
If you found a bug, an improvements, you can report it to GitHub issues: https://github.com/Codeception/CodeceptJS/issues
```

## Type of change

- [ ] Breaking changes
- [ ] New functionality
- [ ] Bug fix
- [ ] Documentation changes/updates
- [ ] Hot fix
- [ ] Markdown files fix - not related to source code

## Checklist:

- [ ] Tests have been added
- [ ] Documentation has been added (Run `robo docs`)
- [x] Lint checking (Run `npm run lint`)
- [x] Local tests are passed (Run `npm test`)

### CODE REVIEW

File: bin/codecept.js [hunk 1]
```diff
@@ -158,16 +158,35 @@ program.command('run-multiple [suites...]')
...
+    info.codeceptVersion = Codecept.version();
+    info.nodeInfo = await envinfo.helpers.getNodeInfo();
+    info.osInfo = await envinfo.helpers.getOSInfo();
+    info.cpuInfo = await envinfo.helpers.getCPUInfo();
+    info.chromeInfo = await envinfo.helpers.getChromeInfo();
+    info.edgeInfo = await envinfo.helpers.getEdgeInfo();
+    info.firefoxInfo = await envinfo.helpers.getFirefoxInfo();
+    info.safariInfo = await envinfo.helpers.getSafariInfo();
+    const fs = require('fs');
+    const config_path = `${process.cwd()}/codecept.conf.js`;
```
Comment (Vorobeyko) [inline bin/codecept.js]: config must be parametrize

File: bin/codecept.js [hunk 2]
```diff
@@ -158,16 +158,35 @@ program.command('run-multiple [suites...]')
...
+    if (fs.existsSync(config_path)) {
+      const { helpers, plugins } = require(`${process.cwd()}/codecept.conf.js`).config;
+      info.helpers = helpers;
```
Comment (Vorobeyko) [inline bin/codecept.js]: What happens if helpers is undefined?

Comment (PeterNgTr) [reply bin/codecept.js]: I thought this but there is no way to have empty helpers there ;)

Comment (Vorobeyko) [reply bin/codecept.js]: I would try to insure against it

File: bin/codecept.js [hunk 3]
```diff
@@ -158,16 +158,35 @@ program.command('run-multiple [suites...]')
 
 program.command('info')
   .description('Print debugging information concerning the local environment')
-  .action(() => {
+  .action(async () => {
+    const info = {};
     console.log('\n Environment information:-\n');
```
Comment (DavertMik) [inline bin/codecept.js]: Move this code to /lib/command/info.js (just like other commands)

File: bin/codecept.js [hunk 4]
```diff
@@ -158,16 +158,35 @@ program.command('run-multiple [suites...]')
...
+    info.osInfo = await envinfo.helpers.getOSInfo();
+    info.cpuInfo = await envinfo.helpers.getCPUInfo();
+    info.chromeInfo = await envinfo.helpers.getChromeInfo();
+    info.edgeInfo = await envinfo.helpers.getEdgeInfo();
+    info.firefoxInfo = await envinfo.helpers.getFirefoxInfo();
+    info.safariInfo = await envinfo.helpers.getSafariInfo();
+    const fs = require('fs');
+    const config_path = `${process.cwd()}/codecept.conf.js`;
+    if (fs.existsSync(config_path)) {
+      const { helpers, plugins } = require(`${process.cwd()}/codecept.conf.js`).config;
```
Comment (DavertMik) [inline bin/codecept.js]: Please, load config the same way as in other scripts. See 

https://github.com/Codeception/CodeceptJS/blob/master/lib/command/dryRun.js#L15

File: bin/codecept.js [hunk 5]
```diff
@@ -158,16 +158,35 @@ program.command('run-multiple [suites...]')
...
+        console.log(`${key}:  ${value[1]}`);
+      } else {
+        console.log(`${key}:  ${JSON.stringify(value, null, ' ')}`);
+      }
+    }
+    console.log('***************************************');
+    console.log(`\nIf you have questions ask them in our Slack: shorturl.at/cuKU8
+    \nOr ask them on our discussion board: https://codecept.discourse.group/
+    \nIf you found a bug, an improvements, you can report it to GitHub issues: https://github.com/Codeception/CodeceptJS/issues`);
   });
```
Comment (DavertMik) [inline bin/codecept.js]: Ask also to include this report when creating an issue on GitHub

File: bin/codecept.js [hunk 6]
```diff
@@ -156,18 +156,39 @@ program.command('run-multiple [suites...]')
 
   .action(require('../lib/command/run-multiple'));
 
-program.command('info')
+program.command('info [path]')
```
Comment (Vorobeyko) [inline bin/codecept.js]: Can we use prarms same as in another commands?
Example, for config use `-c or --config`

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- lib/command/info.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/CodeceptJS/CodeceptJS/pullRequest?prid=4549405&bid=15304778)

**Pull request merged**